projects/igniteui-angular/src/lib/drop-down/drop-down-item.component.ts

```diff
@@ -204,10 +205,48 @@ export class IgxDropDownItemBase {
     templateUrl: 'drop-down-item.component.html'
 })
 export class IgxDropDownItemComponent extends IgxDropDownItemBase {
+    /**
+     * @hidden
+     */
+    protected _isSelected = false;
+
     constructor(
         @Inject(forwardRef(() => IgxDropDownComponent)) public dropDown: IgxDropDownComponent,
-        protected elementRef: ElementRef
+        protected elementRef: ElementRef,
+        protected selectionAPI: IgxSelectionAPIService
     ) {
         super(dropDown, elementRef);
     }
+
+    /**
+     * Sets/Gets if the item is the currently selected one in the dropdown
+     *
+     * ```typescript
+     *  let mySelectedItem = this.dropdown.selectedItem;
+     *  let isMyItemSelected = mySelectedItem.isSelected; // true
+     * ```
+     */
+    get isSelected() {
+        return this._isSelected;
+    }
+    @Input()
+    set isSelected(value: boolean) {
+        if (this.isHeader || this.disabled) {
```

Comment:
Should `disabled` have an effect on programmatic selection? Usually disabled only refers to the user interaction side.

Reply:
fixed. Developers can select disabled items at through code behind now.